cmd/create/operatorroles/by_clusterkey.go

```diff
@@ -303,7 +304,7 @@ func buildCommands(r *rosa.Runtime, env string,
 				return "", err
 			}
 		} else {
-			policyARN = computePolicyARN(r.Creator.AccountID, prefix, operator.Namespace(), operator.Name(), path)
+			policyARN = computePolicyARN(r, r.Creator.AccountID, prefix, operator.Namespace(), operator.Name(), path)
```

Comment:
Since we send `r` to the function already, we can access `r.Creator.AccountID` from within the function.
```suggestion
			policyARN = computePolicyARN(r, prefix, operator.Namespace(), operator.Name(), path)
```